### PR TITLE
docs: release notes for job queue

### PR DESCRIPTION
**Pull request by eecsliu**
## Checklist

- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
See [Release Note](../docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

[DET-1234]: https://determinedai.atlassian.net/browse/DET-1234?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/release-notes/3347-job-queue-write.txt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+:orphan:
+
+**New Features**
+
+-  Add support for dynamic job modification in the job queue. Users can use the WebUI or CLI to
+   change the priority, resource pool, and queue position of jobs without having to cancel and
+   resubmit them. To update jobs with the WebUI, go to the **Job Queue** Section and find the
```
Comment (hamidzr) [inline docs/release-notes/3347-job-queue-write.txt:7]: nit: 
- `Section` lowercase
- with the webui: in the webui or through the webui?

**Review by hamidzr — APPROVED** [review]
:shipit:

File: docs/release-notes/3347-job-queue-write.txt (hunk 1 above)
Comment (eecsliu) [reply docs/release-notes/3347-job-queue-write.txt:7]: good catch - also probably `through` as opposted to `in`

**Comment by hamidzr** [comment]
should we add a k8 caveat?
is it standard parctice to link to our newly added docs

**Comment by gh-determined-ai** [comment]
> should we add a k8 caveat? is it standard parctice to link to our newly added docs

Don't know about the k8s caveat but you shouldn't link to the docs from the release notes. The link will definitely break. You can use words to direct the reader to the general section of the docs.

File: docs/release-notes/3347-job-queue-write.txt [hunk 2]
```diff
@@ -0,0 +1,10 @@
+:orphan:
+
+**New Features**
+
+-  Add support for dynamic job modification in the job queue. Users can use the WebUI or CLI to
```
Comment (gh-determined-ai) [inline docs/release-notes/3347-job-queue-write.txt:5]: "using" instead of "in" or "through".

**Pull request merged**